ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -382,12 +381,34 @@ S3CustomizeFileSystem(WT_STORAGE_SOURCE *storageSource, WT_SESSION *session, con
     std::string objPrefix;
     if ((ret = s3->wtApi->config_get_string(
            s3->wtApi, session, config, "prefix", &objPrefixConf)) == 0)
-        objPrefix = objPrefixConf.str;
+        objPrefix = std::string(objPrefixConf.str, objPrefixConf.len);
     else if (ret != WT_NOTFOUND) {
         std::cerr << "Error: customize_file_system: config parsing for object prefix";
         return (1);
     }
+    /*
+     * Configure the AWS Client configuration. The options that can be set are documented below:
+     * https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/client-config.html
```

Comment:
This URL is not for the CRT client, for instance, I could not find a reference to throughputTargetGbps or partSize. I reckon skip the url.